assignments/r-8.md

```diff
@@ -3,7 +3,7 @@ layout: page
 title: R Assignment 8
 subtitle: Putting It All Together
 language: R
-exercises: ['Combining Basics', 'Scientific 1', 'Scientific 2', 'Graphing 3', 'Statistics 3']
+exercises: ['Size-biased-extinction']
```

Comment:
This will have to be changes to 1) 'Size-biased extinction' or 2) the title you choose based on the comment I provided in the exercise file.